### PR TITLE
Config changes to publish the package to NPM

### PR DESCRIPTION
**Pull request by nagarjun**
This PR introduces the following changes

* Added a GitHub Action to publish the package to NPM whenever a new release is created
* Updated the package.json file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ For a full list of external dependencies please consult `package.json`.
 
 ### NPM
 
-`npm install nightfall-node-sdk`
+`npm install nightfall-js`
```
Comment (nagarjun) [inline README.md:32]: @evanfuller @rssathe how do you guys feel about this name?

Comment (evanfuller) [reply README.md:32]: works for me

File: .github/workflows/publish-to-npm.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm install
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (evanfuller) [inline .github/workflows/publish-to-npm.yml:18]: can you share this secret NPM_TOKEN with a couple of other folks so we have some redundancy?

**Review by evanfuller — APPROVED** [review]

File: .github/workflows/publish-to-npm.yml (hunk 2 above)
Comment (nagarjun) [reply .github/workflows/publish-to-npm.yml:18]: Yeah, I was just waiting for the final decision on the NPM org name in our Slack conversation. Will share on Keybase once we finalise.

File: package.json [hunk 3]
```diff
@@ -12,9 +12,13 @@
     "prepare": "npm run build",
     "test": "jest --coverage"
   },
-  "keywords": [],
+  "keywords": [
+    "data classification",
+    "security"
+  ],
+  "repository": "github:nightfallai/nightfall-nodejs-sdk",
   "author": "Nagarjun Palavalli <nag@nightfall.ai>",
```
[the remark this answers is not in the document]
Comment (nagarjun) [reply package.json:20]: Lol.. valid point but, it's alright. I wouldn't mind a little spam in exchange for NPM fame. :-)

**Pull request merged**